File: app/src/main/java/kr/or/sungrak/cba/cba_retreat/models/MyInfo.java
package kr.or.sungrak.cba.cba_retreat.models;

import com.google.firebase.database.IgnoreExtraProperties;
import com.google.gson.annotations.SerializedName;

// [START blog_user_class]
@IgnoreExtraProperties
public class MyInfo {
    // GBSLevel STAFF = 99;
    @SerializedName("name")
    String name;
    @SerializedName("campus")
    String campus;
    @SerializedName("gbsLevel")
    String gbsLevel;
    @SerializedName("position")
    String position;
    @SerializedName("age")
    String age;
    @SerializedName("mobile")
    String mobile;
    @SerializedName("isLeader")
    boolean isLeader;

    public String getName() {
        return name;
    }

    public String getCampus() {
        return campus;
    }

    public String getGbsLevel() {
        return gbsLevel;
    }

    public String getPosition() {
        return position;
    }

    public String getAge() {
        return age;
    }

    public String getMobile() {
        return mobile;
    }

    public boolean isLeader() {
        return isLeader;
    }
}
// [END blog_user_class]

File: app/src/main/java/kr/or/sungrak/cba/cba_camp/models/MyInfo.kt
package kr.or.sungrak.cba.cba_camp.models

import com.google.gson.annotations.SerializedName


data class MyInfo(
        @SerializedName("memId") val memId: String,
        @SerializedName("name") val name: String,
        @SerializedName("campus") val campus: String,
        @SerializedName("dt_birth") val dt_birth: String,
        @SerializedName("mobile") val mobile: String,
        @SerializedName("uid") val uid: String,
        @SerializedName("grade") val grade: String,
        @SerializedName("department") val department: String,
        @SerializedName("retreatGbsInfo") val retreatGbsInfo: MyRetreatGbsInfo,
        @SerializedName("gbsInfo") val gbsInfo: MyGbsInfo
)

data class MyGbsInfo(
        @SerializedName("gbs") val gbs: String,
        @SerializedName("position") val position: String
)

data class MyRetreatGbsInfo(
        @SerializedName("retreatId") val retreatId: Int,
        @SerializedName("gbs") val gbs: String,
        @SerializedName("position") val position: String
)